index.html

```diff
@@ -38,5 +38,6 @@
       </p>
     </footer>
   </body>
+  <script src="jquery-3.5.1.js"></script>
```

Comment:
To use jQuery, I downloaded the latest version from the jQuery website. I picked the unminified, full-featured version.

Here, it's being loaded before the Book Finder script, because we need jQuery to be loaded first since we're using it inside `script.js`.